app/views/users/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= link_to 'MyProfile', user_path(current_user) %>
+<%= link_to 'Signout', destroy_user_session_path, method: :delete %>
+
+<h1>User profile</h1>
+<div><strong>Firstname: </strong><%= @user.firstname %></div>
```

Comment:
Lets update indentation

Reply:
okay!

Reply:
every div should also be separated by an empty line right?